assets/pages/regular-contributions/components/regularContributionsPayment.jsx

```diff
@@ -77,9 +77,8 @@ function getStatusMessage(
  */
 function RegularContributionsPayment(props: PropTypes, context) {
   let directDebitButton = null;
-  const isDirectDebitEnabled = props.abParticipations.directDebitTest === 'directDebit' || (window.guardian && window.guardian.directDebitEnable);
 
-  if (props.country === 'GB' && isDirectDebitEnabled) {
+  if (props.country === 'GB') {
```

Comment:
Is DD only for the UK (GB) or also for GBP Transaction countries like Gibraltar, etc?

Reply:
It's for anyone with a UK bank account really

Reply:
Cool, in that case, I think the check as it current stands makes sense.